tensorwaves/physics/helicityformalism/kinematics.py

```diff
@@ -199,19 +206,19 @@ def reaction_kinematics_info(self) -> ParticleReactionKinematicsInfo:
         return self._reaction_info
 
     @property
-    def phase_space_volume(self):
+    def phase_space_volume(self) -> float:
         """Get volume of the defined phase space.
 
         Return:
             `float`
         """
         return 1.0
 
-    def is_within_phase_space(self, events):
+    def is_within_phase_space(self, events: np.ndarray) -> Tuple[bool]:
         """Check whether events lie within the phase space definition."""
         raise NotImplementedError("Currently not implemented.")
```

Comment:
Remove the string, the exception class is already clear by itself.

Reply:
Sorry, actually I was referring to `NotImplementedError("Currently not implemented.")` -- the class `NotImplementedError` does not have to be constructed with a string.